pkg/cloud/services/compute/instance.go

```diff
@@ -615,3 +625,31 @@ func getTimeout(name string, timeout int) time.Duration {
 	}
 	return time.Duration(timeout)
 }
+
+// isTrunkExtSupported verifies trunk setup on the OpenStack deployment.
+func (s *Service) isTrunkExtSupported() (trunknSupported bool, err error) {
+	trunkSupport, err := s.networkingService.GetTrunkSupport()
+	if err != nil {
+		return false, fmt.Errorf("there was an issue verifying whether trunk support is available, please disable trunk on your manifests: %v", err)
```

Comment:
nit: `please disable trunk on your manifests` is probably the wrong advice here, as any error returned by this api call is unlikely to be related to trunk support. `Please try again later` would be better advice and we'll do that anyway, so probably just log it.

Reply:
Thanks, fixed.